src/fvtt/page-script.js

```diff
@@ -20,13 +20,22 @@ class FVTTDisplayer {
                 buttons[button]();
             });
         });
-        return this._postChatMessage(html, character, whisper, play_sound);
+        return this._postChatMessage(html, character, whisper, play_sound, attack_rolls, damage_rolls);
     }
 
-    _postChatMessage(message, character, whisper, play_sound = false) {
+    _postChatMessage(message, character, whisper, play_sound = false, attack_rolls, damage_rolls) {
         const MESSAGE_TYPES = CONST.CHAT_MESSAGE_TYPES || CHAT_MESSAGE_TYPES;
+        const pool = new DicePool([...attack_rolls, ...damage_rolls.map(d => d[1])].map(r => r._roll));
```

Comment:
Does it work with the DNDBDice class when using digital dice ?

Reply:
This has been resolved